### ISSUE TITLE
Release MWMBench on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [trillion-labs/gWorld#1]
Hi @jshin49 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2602.01576.

The paper page lets people discuss about your paper and lets them find artifacts about it. It's fantastic to see that you've already hosted the **gWorld-8B** and **gWorld-32B** models on the Hub! You can also claim the paper as yours on Hugging Face, which will show up on your public profile, and add your GitHub and project page URLs to the paper page metadata.

Would you like to host the **MWMBench** dataset you've introduced on https://huggingface.co/datasets?
I noticed in the comments that the evaluation dataset is "coming soon". Hosting it on Hugging Face will give the benchmark more visibility and allow the community to easily load it with:

```python
from datasets import load_dataset

dataset = load_dataset("trillionlabs/MWMBench")
```

If you're interested, you can find a guide on uploading datasets here: https://huggingface.co/docs/datasets/loading. We also offer a [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to explore the data and screenshots directly in the browser.

After it's uploaded, we can link the dataset to the paper page so researchers can find all your artifacts in one place.

Let me know if you're interested or need any guidance!

Kind regards,

Niels

**Comment by sungjunhan-tl** [comment]
Hi Niels,

Thank you so much for reaching out! 

We are planning on releasing the dataset very soon. It will be added to our gWorld collection: https://huggingface.co/collections/trillionlabs/gworld. Since it contains manually collected trajectories, we're currently reviewing everything to ensure there are no license or privacy issues before release. 

Thanks, 

Sungjun

**Comment by Huiimin5** [comment]
Are you planning to release the full training and evaluation pipeline as well?
If so, do you have a rough timeline in mind?

**Comment by sungjunhan-tl** [comment]
We are currently considering releasing a subset of our training set — we'll let you know! The evaluation pipeline is included in the current repository.

**Comment by Huiimin5** [comment]
Will you also release the test set?
